src/components/DefineCompany/OfficeAddresses.vue

```diff
@@ -104,7 +104,7 @@
         </li>
 
         <!--Records Office Address -->
-        <template v-if="entityFilter(EntityTypes.BCOMP)">
+        <template v-if="!entityFilter(EntityTypes.COOP)">
```

Comment:
I think maybe EntityTypes should be replaced by CorpTypeCd ?

Reply:
I agree. EntityTypes is small localized sample of the CorpTypCd. Will go through and replace all instances.

Reply:
Turns out there is quite a few instances 😆 
Will be a min, but in the next commit!

Reply:
It's worth it! 😀

Reply:
totally!

Reply:
Updated in next commit

Reply:
> Turns out there is quite a few instances 😆

Boy, you weren't kidding!